PlastomeRegionBurstAndAlign.py

```diff
@@ -281,57 +281,39 @@ def remove_duplicates(my_dict):
 
 
 # ------------------------------------------------------------------------------#
-## MAIN
+## MAIN HELPER FUNCTIONS
 # ------------------------------------------------------------------------------#
-def main(args):
-    # --------------------------------------------------------------------------#
-    # UNPACKING INPUT PARAMETERS
+def setup_logger(verbose):
+    global log
+    log = logging.getLogger(__name__)
+    log_format = '%(asctime)s [%(levelname)s] %(message)s'
+    log_level = logging.DEBUG if verbose else logging.INFO
+    coloredlogs.install(fmt=log_format, level=log_level, logger=log)
+    return log
+
+def unpack_input_parameters(args):
     in_dir = args.inpd
-    if not os.path.exists(in_dir):
```

Comment:
Checking if the **input** directory exists is an important feature for a user-friendly software. Did you implement this via some other code? If not, please keep it (albeit in a professional way, if you like).

    if not os.path.exists(in_dir):
        logging.critical("Input directory `%s` does not exist." % in_dir)
        raise Exception()